src/worker/predictive-text-studio-worker-impl.ts

```diff
@@ -146,4 +151,12 @@ export class PredictiveTextStudioWorkerImpl
     };
     return this.storage.updateProjectData(storedData);
   }
+
+  saveKMPPackage(KMP: ArrayBuffer): Promise<void> {
+    return this.storage.saveCompiledKMPAsArrayBuffer(KMP);
+  }
```

Comment:
This is a method used exclusively in this class; it can become public later, if needed, but right now, it seems to be an implementation detail.

```suggestion
  private saveKMPPackage(kmp: ArrayBuffer): Promise<void> {
    return this.storage.saveCompiledKMPAsArrayBuffer(kmp);
  }
```